docs/en/query_language/functions/type_conversion_functions.md

```diff
@@ -17,6 +17,85 @@
 
 Converts `value` to [Decimal](../../data_types/decimal.md) of precision `S`. The `value` can be a number or a string. The `S` (scale) parameter specifies the number of decimal places.
 
+## toDecimal*OrNull
+
+Converts an input string to the value of [Nullable(Decimal(P,S))](../../data_types/decimal.md) data type. This family of functions include:
+
+- `toDecimal32OrNull( expr, S)` — Results with Nullable(Decimal32(S)) data type.
+- `toDecimal64OrNull( expr, S)` — Results with Nullable(Decimal64(S)) data type.
+- `toDecimal128OrNull( expr, S)` — Results with Nullable(Decimal128(S)) data type.
+
+These functions should be used instead of `toDecimal*()` functions, if you prefer to get the `NULL` value instead of exception, when input value parsing error.
+
+**Parameters**
+
+- `expr` — Any of ClickHouse [expressions](../syntax.md#syntax-expressions), returning a value of the [String](../../data_types/string.md) data type. ClickHouse expects the textual representation of the decimal number. For example, `'1.111'`.
```

Comment:
* Any of ClickHouse expressions -> expression
* I'd change quotes in example from backticks+single to just double